src/seamless_communication/cards/seamless_expressivity_trainable.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# MIT_LICENSE file in the root directory of this source tree.
+
+name: seamless_expressivity_trainable
+base: unity_nllb-100
+model_arch: expressivity_v2
+char_tokenizer: "https://huggingface.co/facebook/seamless-streaming/resolve/main/spm_char_lang38_tc.model"
+checkpoint: "/fsx-ust/shared/seamless23_oss/m2m_expressive_unity.pt"
```

Comment:
nit: fine with internal, but maybe replace with placeholder when landing in public repo

Reply:
Yes, sure.